File: pkg/pwrgrep/testdata/fixtures/python/swallowed-errors.py
def f(x):
    # ruleid: python-swallowed-errors
    try:
        x()
    except ValueError:
        pass
    # ruleid: python-swallowed-errors
    try:
        x()
    except KeyError:
        raise
    except:
        # nothing to be done
        pass
    # ok: python-swallowed-errors
    try:
        x()
    except KeyError as e:
        raise RuntimeError("missing") from e

File: pkg/pwrgrep/testdata/fixtures/javascript/swallowed-errors.ts
function f(x, promise) {
  try {
    x();
    // ruleid: js-swallowed-errors
  } catch (e) {}
  try {
    x();
    // ruleid: js-swallowed-errors
  } catch {}
  try {
    x();
    // ok: js-swallowed-errors
  } catch (e) {
    throw new Error("failed", { cause: e });
  }
  // ruleid: js-swallowed-errors
  promise.catch(() => {});
}
